src/hooks/VoicePlayer/dux/initialState.ts

```diff
@@ -0,0 +1,34 @@
+export const VoicePlayerStatus = {
+  IDLE: 'IDLE',
+  PREPARING: 'PREPARING',
+  PLAYING: 'PLAYING',
+  PAUSED: 'PAUSED',
+  COMPLETED: 'COMPLETED',
+} as const;
+export type VoicePlayerStatus = typeof VoicePlayerStatus[keyof typeof VoicePlayerStatus];
+
+export type GroupKey = string;
```

Comment:
Can you explain here what are the components of a group key?
comment is enough

Reply:
Ok I will add

Reply:
I move this type definition to the place where the generateGroupKey function is